--- setup.py.orig	2005-07-15 10:33:43.000000000 +0200
+++ setup.py	2005-07-15 10:34:00.000000000 +0200
@@ -65,7 +65,7 @@
                ('share/bkchem/pixmaps', glob.glob( 'pixmaps/*.gif')),
                ('share/bkchem/dtd', glob.glob( 'dtd/*.dtd') + glob.glob( 'dtd/*.xsd')),
                ('share/bkchem/plugins', glob.glob( 'plugins/*.py')+glob.glob( 'plugins/*.xml')),
-               ('share/doc/bkchem', glob.glob( 'doc/*.xml') + glob.glob( 'doc/*.html') + ['README', 'RELEASE', 'INSTALL', 'INSTALL.binary', 'progress.log']),
+               ('share/doc/bkchem', glob.glob( 'doc/*.xml') + glob.glob( 'doc/*.html') + ['README', 'RELEASE', 'INSTALL', 'progress.log']),
                #('share/doc/bkchem/ps', glob.glob( 'doc/ps/*')),
                #('share/doc/bkchem/pdf', glob.glob( 'doc/pdf/*')),
                ('share/doc/bkchem/html', glob.glob( 'doc/html/*')),
