--- bkchem/paper.py.orig	2005-08-04 07:18:12.000000000 +0200
+++ bkchem/paper.py	2005-08-04 07:20:09.000000000 +0200
@@ -1558,8 +1558,11 @@
 
 
   def any_color_to_rgb_string( self, color):
-    r, g, b = map( lambda x: (x < 256 and x) or (x >= 256 and x//256),  self.winfo_rgb( color))
-    return "#%02x%02x%02x" % (r,g,b)
+    if not color:
+      return "none"
+    else:
+      r, g, b = map( lambda x: (x < 256 and x) or (x >= 256 and x//256),  self.winfo_rgb( color))
+      return "#%02x%02x%02x" % (r,g,b)
   
 
 
